### ISSUE TITLE
Data output

### ISSUE THREAD

**Issue by LingQiu** [orobix/retina-unet#22]
Hi,
I have some questions about the data output. I applied your model to another data set ,but I want to know how to achieve Multi-classification task,I hope you can give me some suggestions.Moreover，my data set need information of color,and I don't know how to process it,I hope you can give me guidance.
Any insightful suggestions are very appreciate!

**Comment by dcorti** [comment]
Hi,
By Multi-classification task, you mean you want to label every pixel of the image with more than 2 categories (blood vessel or not in the retina case)? I think this is very interesting but also quite difficult task. For example,  once I thought about trying to identify veins and arteries (which requires the ground truth though) in these images but I have never tried. 
If your data have color, you can feed your colored pictures to the neural network just changing the number of channels in the first layer (from 1 to 3). However, if the color information is not significantly relevant for the task (like in this case), it is better to convert the image to black and white, in order to simply and speed up the learning of the neural network.

**Comment by LingQiu** [comment]
Thanks the reply.
I tried the code from "Fully Convolutional Networks for Semantic Segmentation" on VOC data set(only modified the num_output).It achieved Multi-classification task(no conditional random fields) but the result is unsatisfied.Plus,I have groundtruth.And now I was wondering that modify the number of output just like the way I did from "Fully Convolutional Networks for Semantic Segmentation".So I will try to Multi-classification task based on this project.I'm not sure it's correct.Would you please give me some advice?

**Comment by gingerly** [comment]
To train on grayscale labels (multi-class), can we change the loss function to MMSE and convolve the final layer with 1 filter instead of 2.  Can the output be considered as pixel value?

**Comment by LingQiu** [comment]
@gingerly Thanks the reply.
